src/libraries/System.Private.CoreLib/src/System/ThrowHelper.cs

```diff
@@ -474,6 +474,12 @@ internal static void ThrowArgumentOutOfRangeException_SymbolDoesNotFit()
             throw new ArgumentOutOfRangeException("symbol", SR.Argument_BadFormatSpecifier);
         }
 
+        [DoesNotReturn]
+        internal static void ArgumentOutOfRangeException_Enum_Value()
+        {
+            throw new ArgumentOutOfRangeException("value", SR.ArgumentOutOfRange_Enum);
+        }
```

Comment:
Is this necessary?  Looks like there's only a few call sites, none of which are on a hot path or in a generic context.

Reply:
my goal was to avoid copying code